Dockerfile.osbs

```diff
@@ -86,7 +86,6 @@ RUN INSTALL_PKGS="\
         memcached \
         openssl \
         skopeo \
-        nodejs \
```

Comment:
It looks like we still need `npm` (part of NodeJS) to complete the step where we download static assets for the frontend.

Reply:
I think it's alright to remove it from Dockerfile.osbs but I'll add it back to our regular Dockerfile and defer removing there until the restructuring PR goes in.

Reply:
As long as we don't run any `npm` commands later on in that container build (which it doesn't look like we do), then it should be fine.